docs/advanced_reference/commands/autonomy_service.md

```diff
@@ -11,46 +11,46 @@ This command group consists of a number of functionalities to manage [the life c
 
-`--use-base-sepolia`              
-:   Use the `Base Sepolia` profile to interact with the Autonolas Protocol registry contracts.
+`--use-base-sepolia`
+: Use the Base Sepolia profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-base`                      
-:   Use the `Base` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-base`
+: Use the Base chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-optimistic-sepolia`        
-:   Use the `Optimistic Sepolia` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-optimistic-sepolia`
+: Use the Optimistic Sepolia chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-optimistic`                
-:   Use the `Optimistic` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-optimistic`
+: Use the Optimistic chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-arbitrum-sepolia`          
-:   Use the `Arbitrum Sepolia` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-arbitrum-sepolia`
+: Use the Arbitrum Sepolia chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-arbitrum-one`              
-:   Use the `Arbitrum One` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-arbitrum-one`
+: Use the Arbitrum One chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-chiado`                    
-:   Use the `Chiado` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-chiado`
+: Use the Chiado chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-gnosis`                    
-:   Use the `Gnosis` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-gnosis`
+: Use the Gnosis chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-polygon-mumbai`            
-:   Use the `Polygon Mumbai` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-polygon-mumbai`
+: Use the Polygon Mumbai chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-polygon`                   
-:   Use the `Polygon` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-polygon`
+: Use the Polygon chain profile to interact with the Autonolas Protocol registry contracts.
 
 `--use-ethereum`
-:   Use the `Ethereum` chain profile to interact with the Autonolas Protocol registry contracts.
+: Use the Ethereum chain profile to interact with the Autonolas Protocol registry contracts.
 
-`--use-goerli`                    
-:   Use the `Goerli` chain profile to interact with the Autonolas Protocol registry contracts.
+`--use-goerli`
+: Use the Goerli chain profile to interact with the Autonolas Protocol registry contracts.
```

Comment:
I suggest removing this, along with support, as Goerli has been deprecated.

Reply:
Created an issue for this #2263